r2/r2/lib/utils/reddit_agent_parser.py

```diff
@@ -103,14 +103,24 @@ class RedditIsFunDetector(RedditBrowser):
 
 
 @register_detector
-class RedditAndroidDetector(RedditBrowser):
+class RedditAndroidDetectorOld(RedditBrowser):
```

Comment:
I would rename this `OldRedditAndroidDetector`, then all the detectors end with a common post-fix.